pkg/views/server/configure.go

```diff
@@ -21,26 +23,24 @@ type ServerUpdateKeyView struct {
 
 func ConfigurationForm(config *serverapiclient.ServerConfig) *serverapiclient.ServerConfig {
 	projectStartCommands := view_util.GetJoinedCommands(config.DefaultProjectPostStartCommands)
+	apiPortView := strconv.Itoa(int(config.GetApiPort()))
+	headscalePortView := strconv.Itoa(int(config.GetHeadscalePort()))
+	frpsPortView := strconv.Itoa(int(config.Frps.GetPort()))
 
 	form := huh.NewForm(
 		huh.NewGroup(
 			huh.NewInput().
 				Title("Providers Directory").
 				Value(config.ProvidersDir).
```

Comment:
You're missing the `Directory will be created if it does not exist` description here.